### PR TITLE
GCI 2018 | Students Dashboard with groups

### PR DESCRIPTION
**Pull request by ilyamaier**
- Added students dahsboard that contains groups

## Video
https://youtu.be/-9jfNfp9Bbk

### CODE REVIEW

**Review by nikitadhiman — COMMENTED** [review]
This is great!

**Review by aashutoshrathi — COMMENTED** [review]
That's great work :sparkles: 
Check few comments that I have left.

**Review by prithaupadhyay — APPROVED** [review]
Amazing work @IlyaMaier 🎉🎉

File: src/app/student/student.component.html [hunk 1]
```diff
@@ -1,3 +1,13 @@
-<p>
-  student works!
-</p>
+<app-navbar></app-navbar>
+<div class="body">
+  <div class="row">
+    <img class="avatar" src="https://www.materialui.co/materialIcons/social/person_black_216x216.png">
+    <div class="about">
+      <h2>Ilya Maier</h2>
+      <p>email@email.com</p>
```
[the remark this answers is not in the document]
Comment (IlyaMaier) [reply src/app/student/student.component.html]: Yes. I forgot that. I'll fix it today after school.

[the remark this answers is not in the document]
Comment (IlyaMaier) [reply src/app/student/student.component.html]: Done.

**Review by aashutoshrathi — APPROVED** [review]
LGTM but in case we are of dynamic avatar add an if like if there is no avatar then show default one.

**Comment by IlyaMaier** [comment]
@aashutoshrathi, Done.

**Review by anshumanv — APPROVED** [review]
✨

**Review by aashutoshrathi — APPROVED** [review]
LGTM :100: 
Thanks :smile:

**Comment by IlyaMaier** [comment]
@nikitadhiman , @aashutoshrathi , @prithaupadhyay , @anshumanv , I saved all the images, that i've taken from outside, to assets.

**Pull request closed without merging**